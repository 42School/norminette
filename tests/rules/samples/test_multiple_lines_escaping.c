//\
\ ola \
\ ola \
\ ola \
   \
   \
   \
\
\
\
\
oxi \
eita
