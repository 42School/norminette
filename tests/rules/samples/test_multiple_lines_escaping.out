[36mtest_multiple_lines_escaping.c[0m - [32mIsComment[0m In "GlobalScope" from "None" line 1":
		<COMMENT=//\ ola \ ola \ ola          oxi eita> <NEWLINE>
test_multiple_lines_escaping.c: Error!
Error: INVALID_HEADER       (line:   1, col:   1):	Missing or invalid 42 header
